codequality/checkstyle/checkstyle-test.xml

```diff
@@ -0,0 +1,261 @@
+<?xml version="1.0"?>
```

Comment:
Would have been nice if this could have been refactored as a base config and override config to avoid duplication, but seems there is no standard paved path way currently.

Reply:
Agreed, xml import does not work with Checkstyle documents since both need to be full XML documents